CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing guidelines
+
+Welcome! We're glad to see you here. This short document should give you some hints to get started with contributing to TermPair.
+
+## Getting started
+
+First, fork the repo and clone it to your computer, then read the section you're interested in. 👍
+
+### Contributing Python code
+
+To modify Python code, setup a virtual environment:
+
+```bash
+python -m venv venv
+```
+
+Then activate it with:
+
+```bash
+. venv/bin/activate
+```
+
+You can then install development dependencies using:
+
+```bash
+pip install -e ".[dev]"
+```
+
+### Contributing frontend code
+
+To modify frontend code, get [yarn](https://yarnpkg.com/en/) and run:
+
+```bash
+make build_frontend
+```
+
+You'll need to reload the TermPair server. Stop it with `Ctrl+C` if it's running, then restart it:
+
+```bash
+termpair server
+```
+
+### Contributing documentation
+
+TermPair's documentation is built with [MkDocs](https://www.mkdocs.org) and the [MkDocs Material theme](https://squidfunk.github.io/mkdocs-material/).
+
+To run the documentation site locally, run:
+
+```bash
+mkdocs serve
```

Comment:
I just tried this and it didn't work for me. I'm curious if it worked for you. This is the correct command, but there seems to be a bug in mkdocs or in my configuration of mkdocs.